src/graph/LookupExecutor.cpp

```diff
@@ -676,5 +702,35 @@ bool LookupExecutor::processFinalVertexResult(RpcResponse &rpcResp,
     return true;
 }
 
+Status LookupExecutor::relationalExprCheck(RelationalExpression::Operator op) const {
+    // Compile will fail after added new relational operations.
+    // Need to consider the logic in method 'traversalExpr' after new relational operation added.
+    switch (op) {
+        case RelationalExpression::Operator::EQ :
+        case RelationalExpression::Operator::GE :
+        case RelationalExpression::Operator::GT :
+        case RelationalExpression::Operator::LE :
+        case RelationalExpression::Operator::LT : {
+            return Status::OK();
+        }
+        case RelationalExpression::Operator::NE :
```

Comment:
We don't support NE any more?

Reply:
> We don't support NE any more?

Will need full scan index when a single NE condition occur. We need very complex optimization logic if we want to support NE. I recommend cut this NE in 1.0 .  and you? What do you think？

Reply:
I don't think so.  IMO for NE, we need to scan the whole index.   It is a very import case for index.
And i suggest we disable range scan for string inside graphd.

Reply:
> I don't think so. IMO for NE, we need to scan the whole index. It is a very import case for index.
> And i suggest we disable range scan for string inside graphd.

got, let me improve it.